documentation/decisions/0007-model-resolution.md

```diff
@@ -0,0 +1,113 @@
+# Model Resolution
+
+## Context and Problem Statement
+
+As described in [ADR-0004](0004-urn-as-identifiers.md), URNs are used to identify model elements and
+allow decoupling identification of model elements from resolving them. In contrast to using URLs as
+both identifiers and locators, this allows resolving model elements from different kinds of sources
+in different contexts. To enable interoperable tooling and ensure consistent workflows for users, a
+certain set of guidelines are required. This decision record outlines common concepts about how
+model elements are organized in files and other structures and how model elements should be
+resolved.
+
+## Decision Drivers
+
+* Certain assumptions and workflows specific to tools and projects were established, not all of
```

Comment:
"not all of which are completely consistent" followed by "this includes" could be misread as "these are all non-consistent tools". Perhaps better to just state the positive case as in 
"Certain assumptions and workflows specific to tools and projects were established, which must all be completely consistent in their usage of Aspect Models and ttl files. This includes, but is not limited to:"

Reply:
Perhaps also add simplicity as a driver of the change:
"Certain assumptions and workflows specific to tools and projects were established, which must use Aspect Models and ttl files in a completely consistent and by convention as simple as possible way. Only doing so, development and usage of tools as well as manual searches and manipulations can be achieved efficiently. These tools and projects includes, but is not limited to: